### ISSUE TITLE
Algorithm to manage many neural network systems. 

### ISSUE THREAD

**Issue by sreramk** [Aplokodika/MachineLearning#8]
Given the way that many neural network systems can be designed to screen various properties of the input it receives, we may inter-link them into a network of neural network systems, where the input to the final decisive system is from many other systems. This requires that each of these separate system should receive separate training data-set.

Hence, the final system that provides direct output will be based upon inputs from screened data, screened from various other ANN units for certain properties. This increases performance and accuracy.

**Comment by sreramk** [comment]
This is still an issue. The next work will be on this area.
